LPS-159265 Implement country endpoint for segmentation tests

Luizcarlosqueiroz opened the pull request:
**Ticket**:
[LPS-159265](https://issues.liferay.com/browse/LPS-159265)

**Motivation**:
To create an organization with country (and with region), it's necessary to add these information afterward using the UI.

**Proposed Solution**:
For 'country':
1. Implement [/country/get-country-by-name](http://localhost:8080/api/jsonws?contextName=&signature=%2Fcountry%2Fget-country-by-name-2-long-java.lang.String)
2. Use `_getCountryAbbreviation` implemented using [LPS-163215](https://issues.liferay.com/browse/LPS-163215)

For 'region':
1. Implement [/region/get-regions](http://localhost:8080/api/jsonws?contextName=&signature=%2Fregion%2Fget-regions-3-long-java.lang.String-boolean#serviceResults)

**Usage**:
Call the API using the arguments
```
JSONOrganization.addOrganization(
	organizationCountry = "China",
	organizationName = "organization Name");
```
Instead of using the steps bellow
```
JSONOrganization.addOrganization(organizationName = "organization Name");

ApplicationsMenu.gotoPortlet(
	category = "Users",
	panel = "Control Panel",
	portlet = "Users and Organizations");

Organization.editOrganizationCP(orgName = "organization Name");

Select(
	locator1 = "Select#COUNTRY",
	value1 = "China");

PortletEntry.save();
```

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Luizcarlosqueiroz commented:
Hey @patriciajeanperez !
I've implemented this other API for `country` and `region`, and also added the possibility to use those as arguments for `organizations`. Let me know if there's something you think should be different :)

This is part of a refactor, so there's no hurry to be merged.

cc: @CarlosBrichete
